docs/security-assessment.md

```diff
@@ -46,38 +46,49 @@ flowchart TD
     A(Customer \n Human User) 
     A2(Puris Endpoint App \n Data Provider \n Out of Scope)
     A3(EDC \n Eclipse Data Space Components Connector \n C-X Member \n Out of Scope)
-    
+    A4(Digital Twin Registry)
+
     B(EDC \n ECLIPSE DataSpace Components Connector \n C-X Operator \n Out of Scope)
 
-    C(Vue User Interface)
+    C(React User Interface)
 
     D6(Master Data Controller \n Exposed for external system based on API Keys \n Out of scope)
-    D7(Product Measures Visualization \n Just visualization of the data \n Out of scope - Not yet developped)
-
+ 
     D(Stock View Controller)
     D2(EDC View Controller)
-    D3(Data Request Controller)
+    D3(Data Request Controller - Separtate \n Instance for each controller like \n Stock / Demand / Production / Delivery)
     D4(Data Response Controller)
     D5[(PostgreSQL DB \n Main Database)]
+    D8(Data Twin Registry)
```

Comment:
"Digital Twin Registry (DTR)" instead of "Data Twin Registry"

Reply:
Fixed, thanks for the review.